plugins/tcp.yaml

```diff
@@ -50,6 +50,19 @@ parameters:
       enable_tls:
         equals: true
     advanced_config: true
+  - name: min_version
+    label: "TLS Minimum Version"
+    description: What minimum version of TLS to use
+    type: enum
+    valid_values:
+    - "1.1"
+    - "1.2"
+    - "1.3"
+    default: "1.2"
```

Comment:
Im being picky, but this is technically a breaking change. Because TLS was already supported by this plugin, I think the min version should be equal to that of the operator. This means we need to allow 1.0. https://github.com/observIQ/stanza/blob/master/operator/builtin/input/tcp/tcp.go#L111

The OTEL branch can use 1.2 as the default as that is what the open telemetry implementation is doing: https://pkg.go.dev/go.opentelemetry.io/collector/config/configtls#section-readme